jib-gradle-plugin/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Bug adding base image layers in reverse order and creating wrong image when registry uses V2 image manifest, schema version 1 such as Quay ([#1627](https://github.com/GoogleContainerTools/jib/issues/1627))
```

Comment:
We've been using a format of either `Fixed an issue where ...` or something like `Image manifest V2 Schema 1 requires layers in opposite order to Schema 2`.